Sources/ComposableArchitecture/Store.swift

```diff
@@ -362,15 +362,30 @@ public final class Store<State, Action> {
     self.scope(state: toLocalState, action: { $0 })
   }
 
-  func send(_ action: Action, originatingFrom originatingAction: Action? = nil) {
+  func send(_ action: Action, originatingFrom originatingAction: Action? = nil, instrumentation: Instrumentation = .shared) {
     self.threadCheck(status: .send(action, originatingAction: originatingAction))
 
+    let eventInfo = Instrumentation.EventInfo(
+      type: String(describing: Store.self),
+      action: debugCaseOutput(action),
```

Comment:
all this unsuppressable stringification seems like it'll be a non-starter for upstreaming, but also I hope you can avoid this by passing the data natively instead.